perf_test/blas/blas3/KokkosBlas3_common.hpp

```diff
@@ -139,6 +139,7 @@ static std::string loop_e_str[LOOP_N] = {"serial", "parallel"};
  */
 typedef enum TEST {
   BLAS,
+  BATCHED,
```

Comment:
Is this BATCHED for testing square matrix with heuristic algorithm? Should we rename it to, perhaps, BATCHED_HEURISTIC?